tests/common/libvm.sh

```diff
@@ -347,20 +347,14 @@ vm_assert_status_jq() {
 
 # Like build_rpm, but also sends it to the VM
 vm_build_rpm() {
-    if ! build_rpm "$@" &> build.txt; then
-      cat build.txt
-      assert_not_reached "failed to build RPM"
-    fi
+    build_rpm "$@"
```

Comment:
OK yeah...at some point we should think about a non-shell-script language for this stuff but...